### PR TITLE
add: generate also html coverage reports when running script 'test'

### PR DESCRIPTION
**Pull request by NicolaeIotu**
When running 'composer test', if XDebug is installed and setup (xdebug.mode = coverage), in addition to the file build/logs/clover.xml, generate also nice html coverage reports in folder build/coverage.

PHPunit runs normally if XDebug is missing or it is not set for coverage.

View the coverage report at build/coverage/index.html.

**Description**
Explain what you have changed, and why.

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

**Comment by NicolaeIotu** [comment]
Good idea. Thank you.

**Comment by kenjis** [comment]
I'm fine with this change.
Can you also update the README?
https://github.com/codeigniter4/CodeIgniter4/blob/develop/tests/README.md#generating-code-coverage

**Comment by NicolaeIotu** [comment]
Okay. I'm on it.

File: phpunit.xml.dist [hunk 1]
```diff
@@ -35,6 +35,8 @@
 
         <report>
             <clover outputFile="build/logs/clover.xml"/>
+            <html outputDirectory="tests/coverage" highLowerBound="80"/>
+            <text outputFile="tests/coverage.txt"/>
```
Comment (kenjis) [inline phpunit.xml.dist:39]: I don't like to put files in `tests/` because it is for test code.
Use `build/` instead.

File: system/Pager/PagerRenderer.php [hunk 2]
```diff
@@ -25,14 +25,14 @@
 class PagerRenderer
 {
     /**
-     * First page number.
+     * First page number in the set of links to be displayed.
```
Comment (kenjis) [inline system/Pager/PagerRenderer.php:28]: You seemed to do something wrong with git.
Please get rid of unrelated commits from this PR.

**Comment by NicolaeIotu** [comment]
I will do another PR.
